.github/workflows/ci-executor-unit-test.yml

```diff
@@ -0,0 +1,117 @@
+name: executor-unit-test
+on:
+  schedule:
+    - cron: "40 19 * * *" # Every day starting at 3:40 BJT
+  pull_request_target:
+    paths:
+      - src/promptflow/**
+      - scripts/**
+      - '**ci-executor-unit-test.yml'
+  workflow_dispatch:
+env:
+  packageSetupType: promptflow_with_extra
+  testWorkingDirectory: ${{ github.workspace }}/src/promptflow
+  PYTHONPATH: ${{ github.workspace }}/src/promptflow
+  IS_IN_CI_PIPELINE: "true"
+jobs:
+  authorize:
+    environment:
+      # forked prs from pull_request_target will be run in external environment, domain prs will be run in internal environment
+      ${{ github.event_name == 'pull_request_target' &&
+      github.event.pull_request.head.repo.full_name != github.repository &&
+      'external' || 'internal' }}
+    runs-on: ubuntu-latest
+    steps:
+      - run: true
+  executor_unit_tests:
+    needs: authorize
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
remove macos

Reply:
let's keep macos in this PR, this PR targets to resolve the merge main issue.